site/src/components/Resources/AgentOutdatedTooltip.tsx

```diff
@@ -50,14 +46,17 @@ export const AgentOutdatedTooltip: FC<AgentOutdatedTooltipProps> = ({
     "To fix this, you can stop and start the workspace.";
 
   return (
-    <HelpPopover
-      id={id}
-      open={open}
-      anchorEl={anchorEl}
-      onOpen={onOpen}
-      onClose={onClose}
-    >
-      <HelpTooltipContext.Provider value={{ open, onClose }}>
+    <HelpTooltip>
+      <HelpTooltipTrigger>
+        <span
+          role="presentation"
```

Comment:
Should this have a role of presentation? If I'm understanding things right, all it would do here would make it so that someone with a screen reader can still interact with the trigger, but they'd have no idea what text is inside it (because the role would strip that info from the accessibility tree)

Reply:
I think you could get away with keeping the markup the same, and just removing the `role` prop

Reply:
100%, I changed the role to "status" and removed the aria-label since the text itself is descriptive enough.